test/e2e/specs/editor/various/post-editor-template-mode.spec.js

```diff
@@ -57,6 +57,7 @@ test.describe( 'Post Editor Template mode', () => {
 		);
 
 		// Save changes.
+		await page.click( 'role=button[name="Back"i]' );
```

Comment:
Why is this needed now?

Reply:
Now that we are loading the template into the editor the Post's `save` and `publish` options do not display. To save and publish the post you now need to navigate back to the post.

Reply:
Makes sense.